zerver/openapi/zulip.yaml

```diff
@@ -9170,6 +9211,25 @@ components:
           type: boolean
           description: |
             Whether subdomains are allowed for this domain.
+    RealmPlayground:
+      type: object
+      additionalProperties: false
+      description: |
+        Object containing details about a realm playground.
+      properties:
+        name:
+          type: string
+          description: |
+            The name of the playground.
```

Comment:
I'd expand this slightly more to explain this is how users will be able to select which playground to use if there are multiple options.